src/PrestaShopBundle/Resources/config/services/bundle/command.yml

```diff
@@ -0,0 +1,5 @@
+services:
+    PrestaShopBundle\Command\:
+        resource: "%kernel.root_dir%/../src/PrestaShopBundle/Command/*"
```

Comment:
ping @zuk3975 

@atomiix does this meand that Symfony 4.4 require all services to be tagged 'console.command'? if yes we might need to check how Modules can add their own Commands https://devdocs.prestashop.com/1.7/modules/concepts/commands/ (in another PR)

Reply:
https://github.com/PrestaShop/example-modules/pull/60
I think it doesn't change anything for modules, as I've simply tagged a console command in :arrow_up: example module and it all works like a charm.

btw, Im not sure if this is the right topic, but I don't like that we are getting all services from container straight in our commands, actually all the commands could be registered in services.yml with a `console.command` tag and services can be injected as usual.

Reply:
>btw, Im not sure if this is the right topic, but I don't like that we are getting all services from container straight in our commands, actually all the commands could be registered in services.yml with a console.command tag and services can be injected as usual. (and we could get rid of abstract "ContainerAwareCommand")

@zuk3975 That's exactly what I'm currently working on to get rid of the deprecations!